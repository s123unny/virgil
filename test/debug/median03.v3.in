b ../core/median03.v3 29
run
bt
info l
c
q
