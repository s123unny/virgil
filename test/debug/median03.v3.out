breakpoint|../core/median03.v3 29|0
stop|stopOnBreakpoint
bt|main|../core/median03.v3|29
variable|0|a|0|int|false
variable|1|b|0|int|false
variable|2|c|0|int|false
variable|3|d|0|int|false
variable|4|e|0|int|false
variable|5|median|0|int|false
variable|6|t1|0|int|false
variable|7|t2|0|int|false
stop|end
result|0
